src/DotLiquid/Util/R.cs

```diff
@@ -32,9 +32,7 @@ public static Regex B(string format, params string[] args)
         /// <returns>the regex</returns>
         public static Regex C(string pattern, RegexOptions options = RegexOptions.None)
         {
-#if !CORE
             options = options | RegexOptions.Compiled;
-#endif
             var regex = new Regex(pattern, options, Template.RegexTimeOut);

```

Comment:
Remove previous line and
```suggestion
            var regex = new Regex(pattern, options | RegexOptions.Compiled, Template.RegexTimeOut);
```